benchmarks/cassandra-stress/templates/stress.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ $.Name }}
+  namespace: {{ $.Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: {{ $.Name }}
+      cassandra-stress: {{ $.OperatorName }}
+  serviceName: {{ $.Name }}-stress-svc
+  replicas: {{ .Params.PARALLEL }}
+  podManagementPolicy: Parallel
+  template:
+    metadata:
+      labels:
+        app: {{ $.Name }}
+        cassandra-stress: {{ $.OperatorName }}
+    spec:
+      securityContext:
+        runAsNonRoot: true
+        runAsUser: 999
+        runAsGroup: 999
+        fsGroup: 999
+      containers:
+        - name: cassandra-stress
+          image: mesosphere/cassandra:3.11.6-1.0.0
```

Comment:
We need to ensure that this version gets updated when the version defined in `metadata.sh` is changed. Should this get templatized like it is for the Cassandra operator?

Reply:
Yeah... It probably should. I shy away from templating it though, it feels like an additional step somehow, but I guess it's probably for the best. Still not sure if we should add templating to kuttl, but i guess it needs to be done at some point...